play-json-ops-common/src/main/scala/play/api/libs/json/ops/DurationFormat.scala

```diff
@@ -67,7 +68,17 @@ trait StringDurationFormat extends ImplicitDurationReads with ImplicitDurationWr
  *
  * For example: Duration("1 second") => [1, "SECONDS"]
  */
-trait ArrayDurationFormat extends ImplicitDurationReads with ImplicitDurationWrites with ImplicitTupleFormats {
+trait ArrayDurationFormat extends ImplicitDurationReads with ImplicitDurationWrites {
+
+  override implicit val finiteDurationReads: Reads[FiniteDuration] = new Reads[FiniteDuration] {
+    override def reads(json: JsValue): JsResult[FiniteDuration] = json.validate[(Long, String)] flatMap {
+      case (x, unit) =>
```

Comment:
What is `x`?

Reply:
The scalar value

Reply:
Can we just add types like `case (x: X, unit: Unit) =>`.